docs/features.md

```diff
@@ -0,0 +1,13 @@
+# Features
+
+Some of the key features of Percona Everest are:
+
+- **Multi-Cloud and on-premise deployments**: Percona Everest enables you to transcend the limitations of a single cloud provider. Deploy your databases across multiple clouds or on-premises infrastructure, including OpenShift, to ensure redundancy, high availability, and data sovereignty.
+- **Multiple open source database support**: Percona Everest offers managed solutions for popular open-source databases, including PostgreSQL, MySQL, and MongoDB.
+- **Private deployments and DIY**: With Percona Everest, you'll have complete control over your database environment, ensuring the privacy and security of your data. Private deployments enable you to customize every aspect of your database setup using a do-it-yourself approach.
+- **Database provisioning**
+    * **Scalability**: Your database resources can be scaled to handle varying workloads.
+    * **Database storage class support**: Tailor your storage needs easily by leveraging Percona Everest's database storage class support. Allocate resources efficiently while maintaining optimal performance and cost.
+    * **Disaster recovery capabilities**: Percona Everest prioritizes data protection with a comprehensive disaster recovery suite. Set up backup schedules, create on-demand backups, perform point-in-time recovery, seamlessly restore existing databases, or create new ones from backups.
```

Comment:
we don't have bakcups schedule or point in time recovery yet

Reply:
Fixed.